pytorch_lightning/trainer/trainer.py

```diff
@@ -1669,41 +1667,49 @@ def _on_exception(self) -> None:
 
     @property
     def accelerator(self) -> Accelerator:
-        return self.training_type_plugin.accelerator
+        return self.strategy.accelerator
+
+    @property
+    def strategy(self) -> Strategy:
+        return self._accelerator_connector.strategy
 
     @property
     def training_type_plugin(self) -> Strategy:
-        return self._accelerator_connector.training_type_plugin
+        rank_zero_deprecation(
+            "`Trainer.training_type_plugin` is deprecated in v1.6 and will be removed in v2.0. Use"
```

Comment:
2.0? That sounds extreme 😅

Reply:
1.8 then :)?

Reply:
Yes that would be the first choice imo. 
It was not agreed on what 2.0 means formally, but my understanding was that it will concern api changes that are deeply rooted in how Lightning is used today. This cannot be said about the ttp IMO, which is rarely accessed by the user. The usual 2 version deprecation process should be more than enough :)